packages/twenty-server/src/workspace/workspace-query-builder/factories/delete-many-query.factory.ts

```diff
@@ -8,13 +8,18 @@ import { computeObjectTargetTable } from 'src/workspace/utils/compute-object-tar
 
 import { FieldsStringFactory } from './fields-string.factory';
 
+export interface DeleteManyQueryFactoryOptions
+  extends WorkspaceQueryBuilderOptions {
+  atMost?: number;
```

Comment:
Maybe we want to inject the env variable directly here instead of passing this down from the query-runner, is there any specific reason where this parameter would be overwritten or/and wouldn't come from the env so it would benefit from being a parameter here?

Reply:
@Weiko In my opinion we don't have "reason" to put it as a parameter, but I found this more clean and clear in my opinion.
I don't really like to inject environment service directly into the factories, and if one day we need to create queries based on something else than the env we can :)

Reply:
Ok 👌